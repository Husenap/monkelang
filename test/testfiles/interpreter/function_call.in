monke work sum want number a and number b give number monke do
  give a + b
monke done

monke say "The sum of 3 and 5 is " + monke do sum with 3 and 5 monke done


