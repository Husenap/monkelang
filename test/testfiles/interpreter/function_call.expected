The sum of 3 and 5 is 8
